Observer/Admin/DashboardInstantiation.php

```diff
@@ -125,9 +125,10 @@ public function execute(Observer $observer) : void
                 $store = $this->storeManager->getStore();
             }
             $meta = $this->config->getStoreMetadata($store->getStoreId());
-            if ($meta && ($meta->isActive || $meta->token)) {
+            if ($meta && ($meta->isActive && $meta->token)) {
```

Comment:
I think we need to keep https://github.com/ns8inc/protect-integration-magento/commit/eaca0be6a2f764c72d90ae467a8d07320bbe91fd don't we?